### ISSUE TITLE
go install: version is required when current directory is not in a module

### ISSUE THREAD

**Issue by talentlessguy** [xxxserxxx/gotop#194]
I am trying to install gotop on my new VPS with go 1.16.6 and this is the error I get:

```sh
root@v1rtl:~# go install github.com/xxxserxxx/gotop/v4/cmd/gotop
go install: version is required when current directory is not in a module
```

**Comment by xxxserxxx** [comment]
Yeah, that needs a `@latest` at the end; the `README.md` needs a fix.

**Comment by xxxserxxx** [comment]
Fixed by 257706b0f84c223d4b0c3ca06b22fb5d21b4fefb
